### PR TITLE
webui-Refine the editor

### PR DESCRIPTION
**Pull request by bhaskarmelkani**
- Add front-end validations to editor.

- Make parse yml a safe operation.

- Remove extra line from ace editor.

Relates to #286

### CODE REVIEW

**Comment by rbarilani** [comment]
@bhaskarmelkani let's see tomorrow... because I have also introduced some kind of the validation to hotfix some bugs see #292 and #290

**Comment by bhaskarmelkani** [comment]
👍

File: web-ui/src/client/app/components/editor.jsx [hunk 1]
```diff
@@ -18,13 +18,13 @@ export function ValidateButton (props) {
 export function EditorInputForm (props) {
   return (<form onSubmit={props.onSubmit} className="editor-input-form">
     <label className="dc-label editor-input-form__label">Paste in a Swagger schema and click</label>
-    <ValidateButton disabled={props.pending || props.editorError} />
+    <ValidateButton disabled={props.pending || props.editorError || props.inputBlank} />
```
Comment (rbarilani) [inline web-ui/src/client/app/components/editor.jsx]: @bhaskarmelkani What do you think about ```props.inputIsBlank```?

@bhaskarmelkani can we compute this as:

```js
export function EditorInputForm (props) {
   const validateButtonIsDisabled = props.pending || props.editorError || props.inputBlank;

   return (
    {/* ... */}
    <ValidateButton disabled={validateButtonIsDisabled} />
    {/* ... */}
    )
}
```

Comment (bhaskarmelkani) [inline web-ui/src/client/app/components/editor.jsx]: 👍

**Comment by bhaskarmelkani** [comment]
👍

**Comment by bhaskarmelkani** [comment]
👍

**Comment by rbarilani** [comment]
👍

**Pull request merged**